js/index.js

```diff
@@ -179,6 +179,46 @@
           .format(formatPercent)
       ),
 
+    "cities": renderBlock()
+      .transform(function(d) {
+        var total = d.totals.visits;
+
+        // remove "(not set) from the data"
+        var city_list = d.data.slice(0, 11);
```

Comment:
If I filter the entire list before slicing it adds a couple seconds to the data load time, so I'm filtering from a pre-sliced segment.